### PR TITLE
fix(order.cartServiceOption): duplicate service name

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: src/order/cart/serviceOption/order-cart-serviceOption.v6.service.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/**
+ *  @deprecated
```
Comment (Jisay) [inline src/order/cart/serviceOption/order-cart-serviceOption.v6.service.js]: I think it can be totally removed.
Not only document it as deprecated

Comment (FredericEspiau) [reply src/order/cart/serviceOption/order-cart-serviceOption.v6.service.js]: Okay but we need a major version then :)

Comment (Jisay) [reply src/order/cart/serviceOption/order-cart-serviceOption.v6.service.js]: You're wright it's too dangerous. This solution is the best for the moment and we need to find progressively where the other service is used before remove it.

**Comment by Jisay** [comment]
Duplicate of #132

**Pull request closed without merging**